deps.edn

```diff
@@ -4,142 +4,143 @@
+  bigml/histogram                             {:mvn/version "4.1.3"}              ; Histogram data structure
+  buddy/buddy-core                            {:mvn/version "1.10.1"              ; various cryptograhpic functions
+                                               :exclusions  [commons-codec/commons-codec
+                                                             org.bouncycastle/bcpkix-jdk15on
+                                                             org.bouncycastle/bcprov-jdk15on]}
+  buddy/buddy-sign                            {:mvn/version "3.4.1"}              ; JSON Web Tokens; High-Level message signing library
+  cheshire/cheshire                           {:mvn/version "5.10.1"}             ; fast JSON encoding (used by Ring JSON middleware)
+  clj-http/clj-http                           {:mvn/version "3.12.3"              ; HTTP client
+                                               :exclusions  [commons-codec/commons-codec
+                                                             commons-io/commons-io
+                                                             slingshot/slingshot]}
+  clojurewerkz/quartzite                      {:mvn/version "2.1.0"              ; scheduling library
+                                               :exclusions  [c3p0/c3p0]}
+  colorize/colorize                           {:mvn/version "0.1.1"              ; string output with ANSI color codes (for logging)
+                                               :exclusions  [org.clojure/clojure]}
+  com.cemerick/friend                         {:mvn/version "0.2.3"              ; auth library
+                                               :exclusions  [commons-codec/commons-codec
+                                                             net.sourceforge.nekohtml/nekohtml
+                                                             org.apache.httpcomponents/httpclient
+                                                             ring/ring-core
+                                                             slingshot/slingshot]}
+  com.clearspring.analytics/stream            {:mvn/version "2.9.8"               ; Various sketching algorithms
+                                               :exclusions  [it.unimi.dsi/fastutil
+                                                             org.slf4j/slf4j-api]}
+  com.draines/postal                          {:mvn/version "2.0.4"}              ; SMTP library
+  com.google.guava/guava                      {:mvn/version "30.1.1-jre"}         ; dep for BigQuery, Spark, and GA. Require here rather than letting different dep versions stomp on each other — see comments on #9697
+  com.h2database/h2                           {:mvn/version "1.4.197"}            ; embedded SQL database
+  com.snowplowanalytics/snowplow-java-tracker {:mvn/version "0.10.1"}             ; Snowplow analytics
```

Comment:
This is the only line I've added, but since the package name is quite long I had to re-align all the other lines. Not sure if there's a way to avoid this.

Reply:
its probably best to revert this change and then split the dep and coordinates over two lines
```clojure
  com.h2database/h2                         {:mvn/version "1.4.197"}            ; embedded SQL database
  com.snowplowanalytics/snowplow-java-tracker
                                            {:mvn/version "0.10.1"}
  com.taoensso/nippy                        {:mvn/version "3.1.1"}              ; Fast serialization (i.e., GZIP) library for Clojure
```

The autoformatter will pull the coordinates to the left. If that bothers you you can do something like
```clojure
  com.h2database/h2                         {:mvn/version "1.4.197"}            ; embedded SQL database
  com.snowplowanalytics/snowplow-java-tracker
  #_ -> {:mvn/version "0.10.1"}
  com.taoensso/nippy                        {:mvn/version "3.1.1"}              ; Fast serialization (i.e., GZIP) library for Clojure
```

I think i want the history far more than i want alignment here

Reply:
I like that solution, thanks. Definitely agree about the history